daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Compiler.scala

```diff
@@ -194,6 +194,17 @@ final case class Compiler(packages: PackageId PartialFunction Package) {
   private val compileGetTime: SExpr =
     SEAbs(1) { SBGetTime(SEVar(1)) }
 
+  private val SBLessNumeric =
+    SEAbs(3, SEApp(SEBuiltin(SBLess), Array(SEVar(2), SEVar(1))))
```

Comment:
Is this to "drop" one of the parameters?

Reply:
This drop the type parameter `nat`, which is nor erase.